File: src/main/java/ru/ioleksiv/telegram/bot/api/model/annotation/validator/impl/OrderInfoFilterValidator.java
package ru.ioleksiv.telegram.bot.api.model.annotation.validator.impl;

import ru.ioleksiv.telegram.bot.api.model.annotation.validator.CustomValidator;
import ru.ioleksiv.telegram.bot.api.model.annotation.validator.FilterValidator;
import ru.ioleksiv.telegram.bot.api.model.objects.payments.OrderInfo;

public class OrderInfoFilterValidator extends FilterValidator<OrderInfo> {

    public OrderInfoFilterValidator(CustomValidator<OrderInfo> validator) {
        super(OrderInfo.class, validator);
    }
}

File: core/src/main/kotlin/io/github/oleksivio/tl/kbot/core/model/annotation/validator/impl/OrderInfoFilterValidator.kt
package io.github.oleksivio.tl.kbot.core.model.annotation.validator.impl

import io.github.oleksivio.tl.kbot.core.model.annotation.validator.FilterValidator
import io.github.oleksivio.tl.kbot.server.api.objects.payments.OrderInfo
import io.github.oleksivio.tl.kbot.core.core.controller.handler.check.Validator

class OrderInfoFilterValidator(validator: Validator<OrderInfo>) : FilterValidator<OrderInfo>(
    OrderInfo::class, validator)
